### DISCUSSION TITLE
can'nt work,

### DISCUSSION THREAD

**Discussion by DavidMGT** [tailwindlabs/headlessui#3196] [Help]
when I import "@headlessui/react": "^2.0.3",it just doesnt work .don'nt know why
here is the print

ou may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|     this.notify = (force, sync) => {
|       var _a, _b;
>       const { startIndex, endIndex } = this.range ?? {
|         startIndex: void 0,
|         endIndex: void 0

**Comment by DavidMGT** [comment]
./node_modules/@tanstack/virtual-core/dist/esm/index.js 218:51
Module parse failed: Unexpected token (218:51)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|     this.notify = (force, sync) => {
|       var _a, _b;
>       const { startIndex, endIndex } = this.range ?? {
|         startIndex: void 0,
|         endIndex: void 0

**Comment by franklynbarrera** [comment]
I have the same error :')

**Reply by franklynbarrera** [reply]
I had to downgrade the version
